Demo/Configuration.swift

```diff
@@ -12,9 +12,9 @@ internal enum ConfigurationConstants {
     // swiftlint:disable explicit_acl
 
     /// Please use your own web server between your app and adyen checkout API.
-    static let demoServerEnvironment = DemoServerEnvironment.test
+    static let demoServerEnvironment = DemoServerEnvironment.local
 
-    static let componentsEnvironment = Environment.test
+    static let componentsEnvironment = Environment(baseURL: URL(string: "http://localhost:8080/")!)
```

Comment:
Don't commit this :D

Reply:
you're right, can I keep it until before merging, because I need to test on local environment